spiderbroflatpak.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.StudioPond.Spiderbro
```

Comment:
`studiopond.org` does NOT exist. Yet

> [ X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
Hi! I own studiopond.co.uk. Would co.uk.studiopond.spiderbro be acceptable or should I just go the io.github route? many thanks!

Reply:
whatever you prefer. Just don't make up domains.